crates/bevy_asset/src/assets.rs

```diff
@@ -69,13 +87,26 @@ impl<T: Asset> Assets<T> {
         self.get_handle(id)
     }
 
+    /// Add/modify the asset pointed to by the given handle.
+    ///
+    /// You __must__ use the returned Strong handle. Failing to do so may result in the unexpected release of the asset.
```

Comment:
Must is a strong word. As long as there is another active handle elsewhere, the asset will remain alive. We should soften this language.

Reply:
Guess I took the `must_use` macro a bit too literally. I'll update it